core/frontend/src/IModelApp.ts

```diff
@@ -6,7 +6,10 @@
  * @module IModelApp
  */
 
-const copyrightNotice = 'Copyright © 2017-2022 <a href="https://www.bentley.com" target="_blank" rel="noopener noreferrer">Bentley Systems, Inc.</a>';
+/** @public */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+export const ITWINJS_CORE_VERSION = require("../../package.json").version; // require resolves from the lib/{cjs,esm} dir
```

Comment:
we would need to expose this via a function and add the @extensionApi (for now) tag for it to be added to the extension API (probably should manually add to index.js and index.d.ts in core-extension as well for now)

Reply:
I don't know if we need to expose this directly to Extensions but rather just allow them to specify a minimum in the manifest. That would be the defining contract saying if you could use a certain feature or not.

Reply:
I thought we wanted to expose it for 3.1 though in the short term? If not, then I agree no need to expose

Reply:
It should be needed from the application side in 3.1 but I don't think there is any use-case for it from an Extension perspective.